app/src/main/java/com/hover/stax/hover/AbstractHoverCallerActivity.kt

```diff
@@ -46,7 +46,16 @@ abstract class AbstractHoverCallerActivity : AppCompatActivity(), PushNotificati
     }
 
     fun runSession(account: Account, action: HoverAction) {
-        runSession(account, action, null, account.id) // Constants.REQUEST_REQUEST
+        runSession(account, action, null, account.id)
+    }
+
+    private fun getAccountNameExtra(account: Account, action: HoverAction): HashMap<String, String>? {
+        if (action.requiredParams.contains(ACCOUNT_NAME)) {
+            val extras = HashMap<String, String>()
+            extras[ACCOUNT_NAME] =
```

Comment:
"extras[ACCOUNT_NAME] =" forgot to set a value.

Reply:
This was meant to be removed, it is actually done in `HoverSession`